<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpDialog</class>
 <widget class="QDialog" name="SignUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>421</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>运动员报名表</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog{
background-image: url(:/images/SignupDialog.png);
background-repeat: no-repeat;  /* 防止图片重复 */
border-radius:5px}
</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>122</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;  /* QComboBox的边框 */
border-radius:5px;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_country">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid gray;  /* QComboBox的边框 */
border-radius:5px;
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_sex">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>168</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
border: 2px solid gray;  /* QComboBox的边框 */
border-radius:5px;
	color: rgb(68, 68, 68);
	background-color: rgb(255, 255, 255);
	font: 700 14pt &quot;微软雅黑&quot;;
}
QComboBox::drop-down {
    subcontrol-origin: padding;  /* 设置子控件相对于组合框的内部边距对齐 */
    subcontrol-position: top right;  /* 设置子控件的位置 */
    width: 28px;  /* 按钮宽度 */
    border-left: 2px solid gray;  /* 按钮左侧的分割线 */
    background-color:lightgray;  /* 按钮的背景色 */
	
}
QComboBox QAbstractItemView {
    font: bold 14px;
    color: white;                   /* 项目文字颜色 */
    background-color: blue;         /* 下拉菜单背景颜色 */
    selection-background-color: darkblue;  /* 选中的项目的背景颜色 */
    selection-color: yellow;        /* 选中项目的文字颜色 */
    padding: 5px;                   /* 内边距，确保文字不靠边 */
    border: 2px solid gray;            /* 下拉菜单的边框 */
    border-radius: 5px;                /* 圆角边框 */
    background-color: lightgray;       /* 下拉菜单的背景色 */
    padding: 5px;
    selection-background-color: darkgray;  /* 选中项目的背景色 */
}
</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>16</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>男</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>304</y>
     <width>102</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
stop:0 rgba(233, 233, 233, 255), 
stop:0.35 rgba(220, 220, 220, 255), 
stop:0.5 rgba(213, 213, 213, 255), 
stop:0.65 rgba(230, 230, 230, 255), 
stop:1 rgba(242, 242, 242, 255));
 border: 2px solid rgb(128,126,126);
 border-radius: 5px;
	color: rgb(100, 100, 100);
}
QPushButton:hover{
        background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
stop:0 rgba(250, 250, 250, 255), 
stop:0.5 rgba(255, 255, 255, 255), 
stop:1 rgba(250, 250, 250, 255));
 border: 2px solid rgb(63,140,161);
 border-radius: 5px;
}
 
QPushButton:pressed{
        background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
stop:0 rgba(255, 255, 255, 255), 
stop:0.5 rgba(235, 247, 253, 255), 
stop:1 rgba(213, 237, 250, 255));
 border: 2px solid rgb(3,110,183);
 border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string>📝 确  认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>304</y>
     <width>102</width>
     <height>42</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>100</red>
         <green>100</green>
         <blue>100</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>233</red>
           <green>233</green>
           <blue>233</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.350000000000000">
          <color alpha="255">
           <red>220</red>
           <green>220</green>
           <blue>220</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.500000000000000">
          <color alpha="255">
           <red>213</red>
           <green>213</green>
           <blue>213</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.650000000000000">
          <color alpha="255">
           <red>230</red>
           <green>230</green>
           <blue>230</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>242</red>
           <green>242</green>
           <blue>242</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>68</red>
         <green>68</green>
         <blue>68</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>100</red>
         <green>100</green>
         <blue>100</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>233</red>
           <green>233</green>
           <blue>233</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.350000000000000">
          <color alpha="255">
           <red>220</red>
           <green>220</green>
           <blue>220</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.500000000000000">
          <color alpha="255">
           <red>213</red>
           <green>213</green>
           <blue>213</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.650000000000000">
          <color alpha="255">
           <red>230</red>
           <green>230</green>
           <blue>230</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>242</red>
           <green>242</green>
           <blue>242</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>233</red>
           <green>233</green>
           <blue>233</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.350000000000000">
          <color alpha="255">
           <red>220</red>
           <green>220</green>
           <blue>220</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.500000000000000">
          <color alpha="255">
           <red>213</red>
           <green>213</green>
           <blue>213</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.650000000000000">
          <color alpha="255">
           <red>230</red>
           <green>230</green>
           <blue>230</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>242</red>
           <green>242</green>
           <blue>242</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>100</red>
         <green>100</green>
         <blue>100</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>100</red>
         <green>100</green>
         <blue>100</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>233</red>
           <green>233</green>
           <blue>233</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.350000000000000">
          <color alpha="255">
           <red>220</red>
           <green>220</green>
           <blue>220</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.500000000000000">
          <color alpha="255">
           <red>213</red>
           <green>213</green>
           <blue>213</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.650000000000000">
          <color alpha="255">
           <red>230</red>
           <green>230</green>
           <blue>230</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>242</red>
           <green>242</green>
           <blue>242</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>68</red>
         <green>68</green>
         <blue>68</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>100</red>
         <green>100</green>
         <blue>100</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>233</red>
           <green>233</green>
           <blue>233</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.350000000000000">
          <color alpha="255">
           <red>220</red>
           <green>220</green>
           <blue>220</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.500000000000000">
          <color alpha="255">
           <red>213</red>
           <green>213</green>
           <blue>213</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.650000000000000">
          <color alpha="255">
           <red>230</red>
           <green>230</green>
           <blue>230</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>242</red>
           <green>242</green>
           <blue>242</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>233</red>
           <green>233</green>
           <blue>233</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.350000000000000">
          <color alpha="255">
           <red>220</red>
           <green>220</green>
           <blue>220</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.500000000000000">
          <color alpha="255">
           <red>213</red>
           <green>213</green>
           <blue>213</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.650000000000000">
          <color alpha="255">
           <red>230</red>
           <green>230</green>
           <blue>230</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>242</red>
           <green>242</green>
           <blue>242</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>100</red>
         <green>100</green>
         <blue>100</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>100</red>
         <green>100</green>
         <blue>100</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>233</red>
           <green>233</green>
           <blue>233</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.350000000000000">
          <color alpha="255">
           <red>220</red>
           <green>220</green>
           <blue>220</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.500000000000000">
          <color alpha="255">
           <red>213</red>
           <green>213</green>
           <blue>213</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.650000000000000">
          <color alpha="255">
           <red>230</red>
           <green>230</green>
           <blue>230</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>242</red>
           <green>242</green>
           <blue>242</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>100</red>
         <green>100</green>
         <blue>100</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>100</red>
         <green>100</green>
         <blue>100</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>233</red>
           <green>233</green>
           <blue>233</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.350000000000000">
          <color alpha="255">
           <red>220</red>
           <green>220</green>
           <blue>220</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.500000000000000">
          <color alpha="255">
           <red>213</red>
           <green>213</green>
           <blue>213</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.650000000000000">
          <color alpha="255">
           <red>230</red>
           <green>230</green>
           <blue>230</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>242</red>
           <green>242</green>
           <blue>242</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>233</red>
           <green>233</green>
           <blue>233</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.350000000000000">
          <color alpha="255">
           <red>220</red>
           <green>220</green>
           <blue>220</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.500000000000000">
          <color alpha="255">
           <red>213</red>
           <green>213</green>
           <blue>213</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.650000000000000">
          <color alpha="255">
           <red>230</red>
           <green>230</green>
           <blue>230</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>242</red>
           <green>242</green>
           <blue>242</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>100</red>
         <green>100</green>
         <blue>100</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton 
{
        background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
stop:0 rgba(233, 233, 233, 255), 
stop:0.35 rgba(220, 220, 220, 255), 
stop:0.5 rgba(213, 213, 213, 255), 
stop:0.65 rgba(230, 230, 230, 255), 
stop:1 rgba(242, 242, 242, 255));
 border: 2px solid rgb(128,126,126);
 border-radius: 5px;
	color: rgb(100, 100, 100);
}
.QPushButton:hover{
        background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
stop:0 rgba(250, 250, 250, 255), 
stop:0.5 rgba(255, 255, 255, 255), 
stop:1 rgba(250, 250, 250, 255));
 border: 2px solid rgb(63,140,161);
 border-radius: 5px;
}
 
.QPushButton:pressed{
        background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
stop:0 rgba(255, 255, 255, 255), 
stop:0.5 rgba(235, 247, 253, 255), 
stop:1 rgba(213, 237, 250, 255));
 border: 2px solid rgb(3,110,183);
 border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string>❎ 取  消</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>214</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox
{
    color:rgb(68, 68, 68);
	background-color: rgb(255, 255, 255);
	width: 80px;
	height: 40px;
	font: 700 16pt &quot;Candara&quot;;
     border:2px solid gray;
     border-radius:5px;
     
}
QSpinBox:up-button{
    width:26px;
	border-left: 2px solid #D3D3D3;
	subcontrol-position:right;
	image: url(:/images/plus.png);
}
QSpinBox:down-button{
	border-right: 2px solid #D3D3D3;
	subcontrol-position:left;
	
	image: url(:/images/minus.png);
     width:26px;

}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
